dydx/dydxFormatter/dydxFormatter/dydxFormatter.swift

```diff
@@ -384,20 +384,36 @@ public final class dydxFormatter: NSObject, SingletonProtocol {
      xxxxx.yyyyy
      */
     public func decimalRaw(number: NSNumber?, size: String?) -> String? {
-        return raw(number: number, size: size)?.replacingOccurrences(of: ",", with: ".")
+        raw(number: number, size: size, locale: Locale(identifier: "en-US"))
+    }
+
+    /*
+     xxxxx.yyyyy
+     */
+    public func decimalRaw(number: NSNumber?, digits: Int) -> String? {
+        raw(number: number, digits: digits, locale: Locale(identifier: "en-US"))
     }
 
     /*
      xxxxxx,yyyyy or xxxxx.yyyyy
      */
-    public func raw(number: NSNumber?, digits: Int) -> String? {
+    public func raw(number: NSNumber?, digits: Int, locale: Locale = Locale.current) -> String? {
```

Comment:
it feels unintuitive to have one `raw` function accept locale, and the other `decimalRaw` does not. I might suggest naming the forcing function it to `decimalForApiInput` or `decimalLocalAgnostic`